gr-fosphor.rb

```diff
@@ -17,6 +17,7 @@ def install
       args = %W[
         -DCMAKE_SHARED_LINKER_FLAGS='-Wl,-undefined,dynamic_lookup'
         -DPYTHON_LIBRARY='#{HOMEBREW_PREFIX}/lib/libgnuradio-runtime.dylib'
+        -DFREETYPE2_INCLUDE_DIR_ftheader='#{HOMEBREW_PREFIX}/Cellar/include'
```

Comment:
Try setting the include to `#{Formula["freetype"].include}`.  FreeType is handled in a special way by Homebrew so that it won't collide with the version of FreeType that comes bundled with OSX.